### PR TITLE
Add more copy&paste love to README

### PR DESCRIPTION
**Pull request by taurus-forever**
## Proposal
Update README to provide real copy&paste examples to launch a bundle from zero level.

## Release Notes
Add more copy&paste love to README

## Testing
Tested locally/manually on my K8s.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,74 @@
...
+
+```shell
+juju destroy-model my-postgresql-k8s -y --destroy-storage --force && \
+juju add-model my-postgresql-k8s && juju status
 ```
 
-## Operators included in this bundle
+## Bundle Components
+
+[![PostgreSQL](https://charmhub.io/postgresql-k8s/badge.svg)](https://charmhub.io/postgresql-k8s) [![PgBouncer](https://charmhub.io/pgbouncer-k8s/badge.svg)](https://charmhub.io/pgbouncer-k8s) [![TLS Certificates](https://charmhub.io/tls-certificates-operator/badge.svg)](https://charmhub.io/tls-certificates-operator)
```
Comment (marceloneppel) [inline README.md:62]: Do we have a badge for the edge channel of the PostgreSQL charm? Currently what's being shown is the  version from the stable channel.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,74 @@
...
+
+- [postgresql-k8s](https://charmhub.io/postgresql-k8s): a k8s charm to deploy PostgreSQL.
+- [PgBouncer-k8s](https://charmhub.io/pgbouncer-k8s): a k8s charm to deploy PgBouncer.
+- [tls-certificates-operator](https://charmhub.io/tls-certificates-operator): TLS operator.
```
Comment (marceloneppel) [inline README.md:66]: Maybe add the edge channel query string parameter to these URLs as it was added to the links from the first section of this README?

**Review by WRFitch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (taurus-forever) [reply README.md:62]: I would like to post here edge badges, but charmhub.io produces no difference in:
https://charmhub.io/postgresql-k8s/badge.svg?channel=edge : [PostgreSQL]()
https://charmhub.io/postgresql-k8s/badge.svg : [PostgreSQL]()

In general, it is not well defined when it shows edge and when stable... My guess: the latest updated channel is shown.
Resolved?

File: README.md (hunk 2 above)
Comment (taurus-forever) [reply README.md]: Done.

**Review by marceloneppel — APPROVED** [review]

**Pull request merged**